generator/graphql-kotlin-federation/build.gradle.kts

```diff
@@ -13,6 +13,17 @@ dependencies {
     testImplementation(libs.reactor.core)
     testImplementation(libs.reactor.extensions)
     testImplementation(libs.junit.params)
+    testImplementation("org.junit.jupiter:junit-jupiter")
+    testRuntimeOnly("org.junit.platform:junit-platform-launcher")
+    constraints {
+        implementation("commons-codec:commons-codec") {
```

Comment:
can you define this in libs toml?

so we could just do
```kotlin
    constraints {
        implementation(libs.commons.codec) {
            because("same reason as below")
        }
    }
```

Reply:
Yes, it was possible.